db/migrations/sqls/20200612085103-add-visit-status-up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE scheduled_calls_table ADD status varchar(20);
```

Comment:
What do we think about defaulting the status to 'scheduled' in the schema?

Reply:
Done.